src/main/java/com/slimebot/commands/level/LeaderboardCommand.java

```diff
@@ -0,0 +1,38 @@
+package com.slimebot.commands.level;
+
+import com.slimebot.level.Level;
+import com.slimebot.main.Main;
+import de.mineking.discord.commands.annotated.ApplicationCommand;
+import de.mineking.discord.commands.annotated.ApplicationCommandMethod;
+import de.mineking.discord.commands.annotated.option.Option;
+import net.dv8tion.jda.api.EmbedBuilder;
+import net.dv8tion.jda.api.entities.MessageEmbed;
+import net.dv8tion.jda.api.events.interaction.command.SlashCommandInteractionEvent;
+
+import java.util.List;
+
+@ApplicationCommand(name = "leaderboard", guildOnly = true)
+public class LeaderboardCommand {
+
+
+    @ApplicationCommandMethod
+    public void performCommand(SlashCommandInteractionEvent event, @Option(name = "limit", required = false) Integer limit) {
```

Comment:
Also add minValue = 3 and maxValue = 15. Showing to many levels at a time will probably break the rendering